frappe/public/js/frappe/ui/toolbar/toolbar.js

```diff
@@ -137,11 +137,13 @@ frappe.ui.toolbar.Toolbar = class {
 				__("Generate Tracking URL")
 			);
 
-			if (frappe.perm.has_perm("RQ Job")) {
-				frappe.search.utils.make_function_searchable(function () {
-					frappe.set_route("List", "RQ Job");
-				}, __("Background Jobs"));
-			}
+			frappe.model.with_doctype("RQ Job").then(() => {
+				if (frappe.perm.has_perm("RQ Job", 0, "read")) {
```

Comment:
@surajshetty3416 we have `can_read` somewhere which is available without network request. `with_doctype` will make a network request. Should avoid it here IMO.